### PR TITLE
WIP

### PR DESCRIPTION
**Pull request by xpivarc**

### CODE REVIEW

File: github/ci/prow-deploy/files/jobs/kubevirt/ci-performance-benchmarks/ci-performance-benchmarks-postsubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            
+            ## plot sig-performance 1.25 weekly vm
+            perf-report-creator weekly-graph \
+              --resource=vmi \
+              --weekly-reports-dir=${OUTPUT_DIR}/weekly/periodic-kubevirt-performance-cluster-100-density-test \
+              --plotly-html=true \
+              --metrics-list=vmiCreationToRunningSecondsP50,vmiCreationToRunningSecondsP95,LIST-virtualmachineinstances-count,LIST-pods-count,LIST-nodes-count,LIST-virtualmachineinstancemigrations-count,LIST-endpoints-count,GET-virtualmachineinstances-count,GET-pods-count,GET-nodes-count,GET-virtualmachineinstancemigrations-count,GET-endpoints-count,CREATE-virtualmachineinstances-count,CREATE-pods-count,CREATE-nodes-count,CREATE-virtualmachineinstancemigrations-count,CREATE-endpoints-count,PATCH-virtualmachineinstances-count,PATCH-pods-count,PATCH-nodes-count,PATCH-virtualmachineinstancemigrations-count,PATCH-endpoints-count,UPDATE-virtualmachineinstances-count,UPDATE-pods-count,UPDATE-nodes-count,UPDATE-virtualmachineinstancemigrations-count,UPDATE-endpoints-count
+            
+            # TODO where do we want to put output?
```
Comment (brianmcarey) [inline github/ci/prow-deploy/files/jobs/kubevirt/ci-performance-benchmarks/ci-performance-benchmarks-postsubmits.yaml:65]: Theres a good bit going on here - would it be better to have some publish.sh script in the ci-performance-benchmarks repo that the postsubmit calls?

Comment (xpivarc) [reply github/ci/prow-deploy/files/jobs/kubevirt/ci-performance-benchmarks/ci-performance-benchmarks-postsubmits.yaml:65]: I agree. @alaypatel07 would you take this?

Comment (alaypatel07) [reply github/ci/prow-deploy/files/jobs/kubevirt/ci-performance-benchmarks/ci-performance-benchmarks-postsubmits.yaml:65]: yes, @brianmcarey would it be okay if I maintain the script here https://github.com/kubevirt/project-infra/tree/main/robots/cmd/perf-report-creator along with the scrape.sh

Comment (brianmcarey) [reply github/ci/prow-deploy/files/jobs/kubevirt/ci-performance-benchmarks/ci-performance-benchmarks-postsubmits.yaml:65]: Yes sure - that makes sense to me

**Comment by alaypatel07** [comment]
@xpivarc after this merges https://github.com/kubevirt/project-infra/pull/2943, this PR should be modified as:

1. `cd ci-performance-benchmarks` reference: https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-periodics.yaml#L819 and https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-periodics.yaml#L782-L785
2. `graph.sh .`
3.  `git add -A weekly`

Hope that helps answer the question on how to get the output.

**Comment by xpivarc** [comment]
@alaypatel07 So the only problem I see with this approach is that we want to push the weekly data. The push itself will trigger this postsubmit again. With that in mind, I would rather convert this into periodic to prevent this recursion. WDYT?

**Comment by alaypatel07** [comment]
@xpivarc periodic job works as well, as long as we set it to run the day after scraper job.

**Comment by rthallisey** [comment]
/close
Covered by https://github.com/kubevirt/project-infra/pull/3003

**Pull request closed without merging**